webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala

```diff
@@ -658,8 +661,14 @@ class PolicyWriterServiceImpl(
       // Folders is a map of machine.uuid -> (base_machine_folder, backup_machine_folder, machine)
       for (folder @ NodePromisesPaths(_, baseFolder, newFolder, backupFolder) <- sortedFolder) {
         // backup old promises
-        logger.trace("Backuping old promises from %s to %s ".format(baseFolder, backupFolder))
-        backupNodeFolder(baseFolder, backupFolder)
+        if (backupPoliciesAtEndOfGeneration) {
+          logger.trace("Backuping old policies from %s to %s ".format(baseFolder, backupFolder))
```

Comment:
could you change it to: `s"Backuping old policies from ${baseFolder} to ${backupFolder}"` for consistency ?